mousestyles/dynamics/tests/test_dynamics.py

```diff
@@ -37,3 +41,59 @@ def test_creat_time_matrix():
     # Checking functions output the correct time matrix
     matrix = create_time_matrix(combined_gap=4, time_gap=1, days_index=0)
     assert matrix.iloc[0, 2181] == 1.0

```

Comment:
I think it might be more clear if we could add an explanation about why we want to assert it to be 1? My guess is that the probability should add up to one but if some comments could be added then it would be better.